src/pages/RandomPage.js

```diff
@@ -36,11 +56,13 @@ const RandomPage = ({ characters }) => {
               isbookmarked={false}
             />
             <Button onClick={getRandomCharacter}>Get Random Character</Button>
+            <Button onClick={getRandomAPICharacter}>Get Random API Character</Button>
           </>
         ) : (
           <>
             <QuestionLogo style={{ width: '500px' }} />
             <Button onClick={getRandomCharacter}>Get Random Character</Button>
+            <Button onClick={getRandomAPICharacter}>Get Random API Character</Button>
```

Comment:
You can improve here some lines. You render your `Button` components on condition. But why? They should render when `randomCharacter` is truthy and falsy. The only thing that should change on this condition should be `QuestionLogo`. The buttons are always there. ;-)